vyper/builtin_functions/functions.py

```diff
@@ -726,6 +765,12 @@ def evaluate(self, node):
     def fetch_call_return(self, node):
         raise CompilerPanic("method_id should always be folded")
 
+    def infer_arg_types(self, node):
+        raise CompilerPanic("method_id should always be folded")
```

Comment:
this is OK -- but might lead to problems when we switch the order of constant folding and type checking